packages/sds/src/message_channel/persistent_storage.ts

```diff
@@ -0,0 +1,173 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { Logger } from "@waku/utils";
+
+import { ChannelId, ContentMessage, HistoryEntry } from "./message.js";
+
+const log = new Logger("sds:persistent-storage");
+
+const HISTORY_STORAGE_PREFIX = "waku:sds:history:";
+
+export interface HistoryStorage {
+  getItem(key: string): string | null;
+  setItem(key: string, value: string): void;
+  removeItem(key: string): void;
```

Comment:
what is that? it does not seem to be "History" right? It's supposed to be the interface for local storage, right? Call it `ILocalStorage` then